rear_rider_device/rear_rider_bluetooth_server/src/main.py

```diff
@@ -19,7 +20,26 @@
 AGENT_MANAGER_IFACE = 'org.bluez.AgentManager1'
 AGENT_PATH = '/bluez/simpleagent'
 BLUETOOTH_ALIAS = 'RearRiderPi4'
-
+BLUEZ_DEVICE_1 = 'org.bluez.Device1'
+DBUS_PROPS_IFACE = 'org.freedesktop.DBus.Properties'
+DISCOVERABLE = 1
+"""
+0 = False
+1 = True
+"""
+DISCOVERABLE_TIMEOUT = 0
```

Comment:
does this override the need to change the discoverable timeout in the pi's bluetooth conf file?

Reply:
Yes, the default is 180 seconds (3 minutes). The reason for this override is because currently the user has no way to turn the discoverability back on after the default 3 minutes time out. Which is why the override of 0 (which means no timeout) is needed.

Reply:
NICE